deploy-eks/fb-editor-chart/templates/deployment.yaml

```diff
@@ -245,6 +256,16 @@ spec:
               secretKeyRef:
                 name: fb-editor-secrets-{{ .Values.environmentName }}
                 key: slack_publish_webhook
+          - name: AWS_SES_SECRET_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                name: fb-editor-secrets-{{ .Values.environmentName }}
+                key: aws_ses_secret_access_key
+          - name: AWS_SES_ACCESS_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                name: fb-editor-secrets-{{ .Values.environmentName }}
+                key: aws_ses_access_key_id
```

Comment:
the background task is now a kubernetes cronjob as opposed to a delayed job queue therefore these will not be needed in the worker config.

they can be injected into the config for the cronjob much in the same way the sessions_trim config injects things

Reply:
Good shout. Done.